docs/index.md

```diff
@@ -7,6 +7,7 @@
 - Develop 🚧
   - [Development Setup](/docs/development_setup.md)
   - [Debug](/README.md#debug)
+  - [Environment development](/docs/environment.md)
```

Comment:
you should copy paste that to the readme too.

Reply:
Done